tests/test_items.py

```diff
@@ -492,6 +496,48 @@
     ),
 }
 
+_SOCIAL_MEDIA_POST_MIN_KWARGS: dict = {
+    "url": "https://example.com/viewjob/12345",
+}
+
+_SOCIAL_MEDIA_POST_ALL_KWARGS: dict = {
+    **_SOCIAL_MEDIA_POST_MIN_KWARGS,
+    "postId": "1569750785201676290",
+    "reactions": Reactions(reposts=1, likes=1, dislikes=1),
+    "text": "Join us for the @usertag_1 2022 Coding Contest and stand a chance to win bragging rights plus awesome "
+    "prizes! Register for free - https://hubs.li/Q01mcRlk0 #coding #webdata #dataextraction #webscraping "
+    "#scrapy #python",
```

Comment:
I’m surprised black allows this.

Could you switch to this syntax to make it a bit easier to tell the extra value lines apart from dict keys?

```python
"key": (
    "first line "
    "second line."
),
```